### ISSUE TITLE
Clique sealing failes with "Extra block data too long."

### ISSUE THREAD

**Issue by soc1c** [paritytech/parity-ethereum#10603]
- **Parity Ethereum version**: 2.5.0
- **Operating system**: Linux
- **Installation**: built from source
- **Fully synchronized**: yes
- **Network**: kotti
- **Restarted**: yes

```
10|parity  | 2019-04-21 10:26:54 UTC Starting Parity-Ethereum/v2.5.0-beta-b52ac20-20190408/x86_64-linux-gnu/rustc1.33.0
10|parity  | 2019-04-21 10:26:54 UTC Keys path /home/parity/.local/share/io.parity.ethereum/keys/kotti
10|parity  | 2019-04-21 10:26:54 UTC DB path /home/parity/.local/share/io.parity.ethereum/chains/kotti/db/717b12a127188bca
10|parity  | 2019-04-21 10:26:54 UTC State DB configuration: fast
10|parity  | 2019-04-21 10:26:54 UTC Operating mode: active
10|parity  | 2019-04-21 10:26:55 UTC Configured for Kotti Testnet using Clique engine

10|parity  | 2019-04-21 10:26:58 UTC Listening for new connections on 127.0.0.1:8546.
10|parity  | 2019-04-21 10:27:03 UTC Public node URL: enode://06333009fc9ef3c9e174768e495722a7f98fe7afd4660542e983005f85e556028410fd03278944f44cfe5437b1750b5e6bd1738f700fe7da3626d52010d2954c@10.0.0.4:30303
10|parity  | 2019-04-21 10:27:04 UTC ERROR: seal failed when given internally generated seal: Block error (Extra block data too long. Value 31 out of bounds. Min=32, Max=32)
10|parity  | 2019-04-21 10:27:06 UTC ERROR: seal failed when given internally generated seal: Block error (Extra block data too long. Value 31 out of bounds. Min=32, Max=32)
10|parity  | 2019-04-21 10:27:08 UTC ERROR: seal failed when given internally generated seal: Block error (Extra block data too long. Value 31 out of bounds. Min=32, Max=32)
10|parity  | 2019-04-21 10:27:10 UTC ERROR: seal failed when given internally generated seal: Block error (Extra block data too long. Value 31 out of bounds. Min=32, Max=32)
10|parity  | 2019-04-21 10:27:12 UTC ERROR: seal failed when given internally generated seal: Block error (Extra block data too long. Value 31 out of bounds. Min=32, Max=32)
10|parity  | 2019-04-21 10:27:14 UTC ERROR: seal failed when given internally generated seal: Block error (Extra block data too long. Value 31 out of bounds. Min=32, Max=32)
10|parity  | 2019-04-21 10:27:16 UTC ERROR: seal failed when given internally generated seal: Block error (Extra block data too long. Value 31 out of bounds. Min=32, Max=32)
10|parity  | 2019-04-21 10:27:18 UTC ERROR: seal failed when given internally generated seal: Block error (Extra block data too long. Value 31 out of bounds. Min=32, Max=32)
10|parity  | 2019-04-21 10:27:20 UTC ERROR: seal failed when given internally generated seal: Block error (Extra block data too long. Value 31 out of bounds. Min=32, Max=32)
10|parity  | 2019-04-21 10:27:22 UTC ERROR: seal failed when given internally generated seal: Block error (Extra block data too long. Value 31 out of bounds. Min=32, Max=32)
10|parity  | 2019-04-21 10:27:24 UTC ERROR: seal failed when given internally generated seal: Block error (Extra block data too long. Value 31 out of bounds. Min=32, Max=32)
10|parity  | 2019-04-21 10:27:26 UTC ERROR: seal failed when given internally generated seal: Block error (Extra block data too long. Value 31 out of bounds. Min=32, Max=32)
10|parity  | 2019-04-21 10:27:28 UTC ERROR: seal failed when given internally generated seal: Block error (Extra block data too long. Value 31 out of bounds. Min=32, Max=32)
10|parity  | 2019-04-21 10:27:30 UTC ERROR: seal failed when given internally generated seal: Block error (Extra block data too long. Value 31 out of bounds. Min=32, Max=32)
10|parity  | 2019-04-21 10:27:32 UTC ERROR: seal failed when given internally generated seal: Block error (Extra block data too long. Value 31 out of bounds. Min=32, Max=32)
10|parity  | 2019-04-21 10:27:33 UTC    4/25 peers     50 KiB chain   53 MiB db  0 bytes queue   15 KiB sync  RPC:  0 conn,    5 req/s,   17 µs
10|parity  | 2019-04-21 10:27:34 UTC ERROR: seal failed when given internally generated seal: Block error (Extra block data too long. Value 31 out of bounds. Min=32, Max=32)
10|parity  | 2019-04-21 10:27:36 UTC ERROR: seal failed when given internally generated seal: Block error (Extra block data too long. Value 31 out of bounds. Min=32, Max=32)
10|parity  | 2019-04-21 10:27:38 UTC ERROR: seal failed when given internally generated seal: Block error (Extra block data too long. Value 31 out of bounds. Min=32, Max=32)
10|parity  | 2019-04-21 10:27:40 UTC ERROR: seal failed when given internally generated seal: Block error (Extra block data too long. Value 31 out of bounds. Min=32, Max=32)
10|parity  | 2019-04-21 10:27:42 UTC ERROR: seal failed when given internally generated seal: Block error (Extra block data too long. Value 31 out of bounds. Min=32, Max=32)
10|parity  | 2019-04-21 10:27:44 UTC ERROR: seal failed when given internally generated seal: Block error (Extra block data too long. Value 31 out of bounds. Min=32, Max=32)
10|parity  | 2019-04-21 10:27:46 UTC ERROR: seal failed when given internally generated seal: Block error (Extra block data too long. Value 31 out of bounds. Min=32, Max=32)
10|parity  | 2019-04-21 10:27:48 UTC ERROR: seal failed when given internally generated seal: Block error (Extra block data too long. Value 31 out of bounds. Min=32, Max=32)
10|parity  | 2019-04-21 10:27:50 UTC ERROR: seal failed when given internally generated seal: Block error (Extra block data too long. Value 31 out of bounds. Min=32, Max=32)
10|parity  | 2019-04-21 10:27:52 UTC ERROR: seal failed when given internally generated seal: Block error (Extra block data too long. Value 31 out of bounds. Min=32, Max=32)
10|parity  | 2019-04-21 10:27:54 UTC ERROR: seal failed when given internally generated seal: Block error (Extra block data too long. Value 31 out of bounds. Min=32, Max=32)
10|parity  | 2019-04-21 10:27:56 UTC ERROR: seal failed when given internally generated seal: Block error (Extra block data too long. Value 31 out of bounds. Min=32, Max=32)
10|parity  | 2019-04-21 10:27:58 UTC ERROR: seal failed when given internally generated seal: Block error (Extra block data too long. Value 31 out of bounds. Min=32, Max=32)
```

**Comment by soc1c** [comment]
cc @niklasad1 @jleni @jwasinger

**Comment by jwasinger** [comment]
I was not voted in as a sealer so I cannot easily test the fix for this.  Can you give it a try and let me know @soc1c ?

**Comment by HCastano** [comment]
Closed by #10605.
